blogger/blogger.php

```diff
@@ -115,18 +116,18 @@ function blogger_settings(&$a, &$s)
 }
 
 
-function blogger_settings_post(&$a, &$b)
+function blogger_settings_post(App $a, array &$b)
 {
-	if (x($_POST, 'blogger-submit')) {
-		PConfig::set(local_user(), 'blogger', 'post', intval($_POST['blogger']));
+	if (isset($_POST['blogger-submit'])) {
```

Comment:
The exact equivalent to `x($_POST, 'blogger-submit')` is `!empty($_POST['blogger-submit'])`.

Reply:
Done.

Reply:
Aren't you supposed to sleep? 😅